srcpkgs/wxWidgets/template

```diff
@@ -16,17 +16,27 @@ homepage="https://www.wxwidgets.org/"
 distfiles="https://github.com/wxWidgets/wxWidgets/releases/download/v${version}/wxWidgets-${version}.tar.bz2"
 checksum=96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0
 
+post_install() {
+	# remove files that are already in wxWidgets-common
+	rm -f ${DESTDIR}/usr/lib/libwx_baseu*
+	rm -rf ${DESTDIR}/usr/bin/wxrc*
+	rm -rf ${DESTDIR}/usr/share
+	find ${DESTDIR}/usr/include -type f -not -name tabartgtk.h \
+		-not -name fontdlgg.h -not -name notifmsg.h -exec rm -f {} \;
```

Comment:
That leaves 
```
/usr/include/wx-3.0/wx/generic/notifmsg.h
/usr/include/wx-3.0/wx/gtk/notifmsg.h
/usr/include/wx-3.0/wx/notifmsg.h
```
in the wxWidgets package which are also in wxWidgets-common-devel.

Can only think of sth. like
```
for f in "" generic gtk; do
	rm ${DESTDIR}/usr/include/wx-3.0/wx/$f/notifmsg.h
done
```
to remove them, but it isn’t really nice. ;)

Reply:
I dind't notice that. Thanks. I will rework that as soon as I am done with the -gtk3 template.